src/task/MailApp.java

```diff
@@ -1,17 +1,101 @@
 package task;
 
+import java.util.*;
```

Comment:
try doesn't use auto import)

Reply:
I'm sorry, but it must be that for verification on the Stepik.

Reply:
Thank you!